languages/en_us.js

```diff
@@ -42,8 +42,13 @@
 			ERROR_AGE_WRONG: 'The age must be valid',
 			ERROR_TERMS_EMPTY: 'You must accept the terms and conditions of the service',
 			ERROR_RECAPTCHA_NOT_CHECKED: 'This app only is able to be used by humans. Aren\'t you human?',
+			GALLERY_FEATURED_VIDEOS: 'Featured videos',
+			GALLERY_VIEW_ALL: 'View all',
+			GALLERY_ORDER_BY: 'Order by',
+			GALLERY_ORDER_BY_SELECTOR_RELEVANCE: 'Relevance',
+			VIDEO_CARD_VERIFIED: 'Verified',
+			VIDEO_CARD_NOT_VERIFIED: 'Not verified',
 			DOWNLOAD_CODE: 'Enter your download code',
-			VERIFIED: 'Verified video',
```

Comment:
What happens with [video-download view (line 26)](https://github.com/Videona/platform-frontend/blob/dev/pages/video-download/video-download.view.html#L26) when you remove this line?

Reply:
As I've seen, that deleted line is correct, but what is not correct and you're right is the calling to the correct variable in the view. At this point, we've changed VERIFIED to VIDEO_CARD_VERIFIED, so I'm updating the view with this name.

Reply:
So, if there's not only being calling in `video-card` component... Shall the name be `VIDEO_CARD_VERIFIED` or better something just like `VIDEO_VERIFIED`?